Rakefile

```
@@ -2,6 +2,7 @@ require "rspec/core/rake_task"
 
 RSpec::Core::RakeTask.new(:spec)
 
+desc "Lint ruby and JavaScript files"
```

Probably should title case Ruby like JavaScript

```suggestion
desc "Lint Ruby and JavaScript files"
```